packages/aws_common/test/io/aws_file_io_test.dart

```diff
@@ -31,32 +31,45 @@ void main() {
     }
     final testFilePath = '$currentPath/test/io/assets/test_file.txt';
     const testFileContent = 'I ❤️ Amplify, œ 小新';
+    const testContentType = 'text/plain';
     final testBytes = utf8.encode(testFileContent);
 
     final testFile = io.File(testFilePath);
 
     group('getChunkedStreamReader() API', () {
       test('should return ChunkedStreamReader over io File', () async {
-        final awsFile = AWSFilePlatform.fromFile(testFile);
+        final awsFile = AWSFilePlatform.fromFile(
```

Comment:
question: Nice assertion here, but would it make sense to add similar tests for other common file formats like images? same question applies to _html version.